src/types.ts

```diff
@@ -39,3 +43,12 @@ export type QuizAtomType = {
 export type TimelineAtomType = {
     id: string;
 };
+
+export type AtomEmbedBlockElement = {
+    _type: string;
+    css: string;
+    js: string;
+    html: string;
+    id: string;
+    url: string;
```

Comment:
Realised we might be missing shouldFence but don't think we need his right now as we'll 'fence' (Stick in an iframe) all atoms.

Reply:
I am not quite sure what you mean by `shouldFence`? are you referring to `sandbox` attr in iframe?

Reply:
No, sorry, I'm referring to this: https://github.com/guardian/frontend/blob/master/common/app/views/fragments/atoms/interactive.scala.html#L36

Basically we 'fence' - iframe - the interativeAtom if on an article page, but otherwise we place it straight into the page. We don't need to worry about it for now though.

Reply:
im curious to know were the variable `shouldFence` comes from? should I add it as part of the fixture?

Reply:
I wouldn't worry for now, it comes via CAPI, but is not in DCRModel yet, so when we need it we can add it.